taggit/templates/admin/taggit/merge_tags_form.html

```diff
@@ -0,0 +1,29 @@
+<div id="mergeTagsModal">
```

Comment:
Maybe `{% extends "admin/base_site.html" %}`, then surround all of this with a `{% block content %}/ {% endblock %}`, and the form will look a _bit_ more like an admin page

Reply:
actually maybe just `admin/base.html`.

Reply:
Updated this and also changed the button to an input to make the styling a little cleaner